### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by serg121192**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+        if isinstance(other, int) or isinstance(other, float):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other: int | float) -> Distance:
+        if isinstance(other, int) or isinstance(other, float):
+            return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Distance | int | float) -> bool:
+        if isinstance(self, Distance) and isinstance(other, Distance):
+            return True if self.km < other.km else False
```
Comment (Abnormaltype) [inline app/main.py:38]:
```suggestion
            return self.km < other.km
```
fix it in other cases

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (serg121192) [reply app/main.py:38]: > fix it in other cases

Fixed) And could you check my solution of decorators?)

Comment (Abnormaltype) [reply app/main.py:38]: check guidelines on how to do a re-request review.

**Review by Vasyl-Smutok — APPROVED** [review]
Good Job 👍

**Pull request closed without merging**